src/App/Pages/Vault/GroupingsPage/GroupingsPageListGroup.cs

```diff
@@ -26,11 +26,13 @@ public GroupingsPageListGroup(string name, int count, bool doUpper = true, bool
             }
             ItemCount = count.ToString("N0");
             First = first;
+            AutomationId = $"{NameShort}ListGroup";
         }
 
         public bool First { get; set; }
         public string Name { get; set; }
         public string NameShort => string.IsNullOrWhiteSpace(Name) || Name.Length == 0 ? "-" : Name[0].ToString();
         public string ItemCount { get; set; }
+        public string AutomationId { get; set; }
```

Comment:
You can remove the public setter here. It's being set in the constructor so you don't need the `set` at all and can directly be:
```c#
public string AutomationId => $"{NameShort}ListGroup";
```